src/ConservationChecker.jl

```diff
@@ -0,0 +1,331 @@
+#=
+    ConservationChecker
+
+This module contains functions that check global conservation of energy and water (and momentum - TODO).  
+=#
+module ConservationChecker
+
+using ClimaCoupler.Utilities
+using ClimaCore: ClimaCore, Geometry, Meshes, Domains, Topologies, Spaces, Fields, InputOutput
+using ClimaCore.Utilities: half
```

Comment:
I think this is redundant since you already have using `ClimaCoupler.Utilities`

Reply:
Actually the half needs to be specifically imported, but line 8 can be removed.